### PR TITLE
[FEA-792] added multiplier

### PR DESCRIPTION
**Pull request by Purva-Chaudhari**
## What's new in this PR?

Add multiplier for flight class

### CODE REVIEW

**Comment by Purva-Chaudhari** [comment]
ran forge fmt

File: plume/src/Faucet.sol [hunk 1]
```diff
@@ -253,14 +258,39 @@ contract Faucet is Initializable, UUPSUpgradeable {
...
+        } else if (flightClass == 2) {
+            multiplier = 11; // 1.1x (scaled by 10)
+        } else if (flightClass == 3) {
+            multiplier = 125; // 1.25x (scaled by 100)
+        } else if (flightClass == 4) {
+            multiplier = 200; // 2x (scaled by 100)
+        } else if (flightClass == 5) {
+            multiplier = 300; // 3x (scaled by 100)
+        } else if (flightClass == 6) {
+            multiplier = 500; // 5x (scaled by 100)
```
Comment (eyqs) [inline plume/src/Faucet.sol:289]: Add comments on what each flight class is (Economy, Plus, etc.)

Comment (eyqs) [reply plume/src/Faucet.sol:289]: There must be a more efficient way of doing this? e.g. `uint256[] constant multipliers = [0, 1, 11, 125, 200, 300, 500];` - check if this actually saves gas

**Review by eyqs — APPROVED** [review]
Can you install the latest version of Foundry and run `forge fmt` again?

**Comment by Purva-Chaudhari** [comment]
made it to array, for 6 classes it's better 

Upgraded forge

**Review by eyqs — APPROVED** [review]
LGTM

File: plume/src/Faucet.sol (hunk 1 above)
Comment (ungaro) [reply plume/src/Faucet.sol:289]: maybe something like this.
```
uint256[6] memory multipliers = [100, 110, 125, 200, 300, 500];
    
if (flightClass == 0 || flightClass > 6) {
    revert InvalidFlightClass(flightClass);
}
    
return (baseAmount * multipliers[flightClass - 1]) / 100;```

**Comment by Purva-Chaudhari** [comment]
Class 6 if else is better, Class 9 and above array works better
image

**Pull request merged**